media-sound/gmusicbrowser/gmusicbrowser-1.1.16.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils gnome2-utils xdg
```

Comment:
Does it really need eutils? If not, remove it

Reply:
yep, `einstalldocs` is used

Reply:
I see the eclass marked as deprecated now. So if you suggest alternative I can drop it.

Reply:
einstalldocs is available in EAPI 6-7 without needing to inherit eutils, the eclass omits defining it if not EAPI 5 or lower

Reply:
Thank you for the hint.
I've removed it.